### ISSUE TITLE
FR: allow to specify where to store favorites

### ISSUE THREAD

**Issue by GitMensch** [leftstick/vscode-favorites#61]
With `favorites.saveSeparated` there is an extra file instead of the configuration settings (at least I guess so).

This feature request is for a new setting `favorites.saveSeparatedAs` or similar, which allows to specify a path to store them, ideally both with a relative path, a full path and resolving [a subset of] [vscode variables](https://code.visualstudio.com/docs/editor/variables-reference), especially `{env:name}` and `{config:name}`.

**Comment by dimateos** [comment]
Yeah a custom path that could parse `${workspaceRoot}` would be great!
I basically need to it inside the .vscode folder e.g. `${workspaceRoot}/.vscode/favorites.json`

**Comment by leftstick** [comment]
Hello @GitMensch,

Thank you for your feature request. Currently, the extension doesn't support customizing the location of the favorites storage file. The `favorites.saveSeparated` setting only allows storing favorites in a .vscfavoriterc file in the workspace root.

I appreciate your interest in this feature. If you'd like to implement it, please feel free to submit a pull request.

Best regards,
The vscode-favorites team
